### ISSUE TITLE
Enumerated code blocks are partially escaped when including empy lines between code

### ISSUE THREAD

**Issue by RoboMarth** [showdownjs/showdown#494]
**Description**: Code blocks that are enumerated (indented so as to be part of list) are partially escaped when there are empty lines between code.

**Input**:

```markdown
1. Code block as part of list

    ```
    public static void main(String[] args) {

        for (int i = 0; i < 10 && true; i++) {

            System.out.println("Hello World");
        }

        // stuff here is affected as well <>&&%
    }
    ```
```

**Expected Output**:

```html
<ol>
<li><p>Code block as part of list</p>
<pre><code>public static void main(String[] args) {

    for (int i = 0; i &lt; 10 &amp;&amp; true; i++) {
    
        System.out.println("Hello World");
    }

    // stuff here is affected as well &lt;&gt;&amp;&amp;%
}
</code></pre></li>
</ol>
```

**Output**:

```html
<ol>
<li><p>Code block as part of list</p>
<pre><code>public static void main(String[] args) {<pre><code>for (int i = 0; i &amp;lt; 10 &amp;amp;&amp;amp; true; i++) {

    System.out.println("Hello World");
}

// stuff here is affected as well &amp;lt;&amp;gt;&amp;amp;&amp;amp;%
</code></pre>}
</code></pre></li>
</ol>
```

**Comment by tivie** [comment]
confimed!

**Comment by KirkMunro** [comment]
@RoboMarth Thanks for logging this, I just ran into the same issue myself. :)

**Comment by James-Burnham** [comment]
Any outlook on this? Would be very excited to see a fix.

**Comment by tivie** [comment]
This shouldbe fixed for version 2.0. Unfortunately I cannot backport for version 1.x

The problem is that showdown thinks that those are 2 codeblocks instead of one...

**Comment by tivie** [comment]
this nasty bug should be fixed now
